components/Navigation/Navigation.js

```diff
@@ -22,18 +18,12 @@ export default function Navigation({currentPage, setCurrentPage}) {
               <path
                 d="M31 62V55.1111H55.1111V6.88889H31V0H55.1111C57.0056 0 58.6279 0.673963 59.9781 2.02189C61.326 3.37211 62 4.99444 62 6.88889V55.1111C62 57.0056 61.326 58.6279 59.9781 59.9781C58.6279 61.326 57.0056 62 55.1111 62H31ZM24.1111 48.2222L19.375 43.2278L28.1583 34.4444H0V27.5556H28.1583L19.375 18.7722L24.1111 13.7778L41.3333 31L24.1111 48.2222Z"
                 // currentPage Indicator
-                fill={currentPage === "login" ? "#12263A" : "#FBF5F3"}
+                fill={router.pathname === "/" ? "#12263A" : "#FBF5F3"}
               />
             </svg>
           </StyledLink>
 
-          <StyledLink
-            onClick={() => setCurrentPage("home")}
-            href={"/home"}
-            className={
-              currentPage === "home" ? "highlightedIcon" : "normalIcon"
-            }
-          >
+          <StyledLink onClick={() => router.push("/home")} href={"/home"}>
             <svg
               width="62"
```

Comment:
Please add a `<title>` tag to your SVGs to make them accessible (https://css-tricks.com/accessible-svgs/).
You can test accessibility with the Lighthouse tool in the Chrome Dev Tools.

Reply:
Hey Ralf, thanks for the Review. I updated it.